### ISSUE TITLE
JSON casing breaking change 4.1

### ISSUE THREAD

**Issue by DanFTRX** [dvsekhvalnov/jose-jwt#217]
https://github.com/dvsekhvalnov/jose-jwt/commit/13f25f9cd67324a134e1b87b31c305b30729591c added camel case overrides on all json serialization in the library.  This is a change from before when it would preserve the casing provided.  It's also not easily worked around as the JsonSerializerOptions are private in the JsonMapper class.

**Comment by dvsekhvalnov** [comment]
Hey @DanFTRX , 

i guess it was intentional, can't recall why exactly today, but probably there were a reasons. 

I'm curious to see unit tests that were working on v4.0 but broken now on 4.1?

**Comment by DanFTRX** [comment]
My problem is that I have a different language client with a case sensitive JSON parser (I could write you a unit test showing you that a case sensitive parser won't work with the new casing, but I'm not sure that will be useful).  One could argue that case sensitivity on the client was a mistake, but I can't change the past and in the end a change to the library has broken my workflow.  I understand the new default may be intentional, but can you provide the ability to modify the settings on the JsonMapper class (or a constructor that takes in a SerializationSettings and DeserializationSettings rather than the defaults) to allow easy modification of this new behavior without having to recreate the entire class?

**Comment by dvsekhvalnov** [comment]
Yeah, can do it, seems to be same request as https://github.com/dvsekhvalnov/jose-jwt/issues/214

One thing is, i'm not sure what will break if you change default casing. May be more viable option for you will be to use methods that returning bare `string` and handle json serialization yourself behind library?
